flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/utils/SimpleAckingTaskManagerGateway.java

```diff
@@ -44,23 +47,21 @@
 
 	private final String address = UUID.randomUUID().toString();
 
-	private Optional<Consumer<ExecutionAttemptID>> optSubmitConsumer;
+	private Consumer<TaskDeploymentDescriptor> submitConsumer = ignore -> { };
 
-	private Optional<Consumer<ExecutionAttemptID>> optCancelConsumer;
+	private Consumer<ExecutionAttemptID> cancelConsumer = ignore -> { };
 
 	private volatile BiFunction<AllocationID, Throwable, CompletableFuture<Acknowledge>> freeSlotFunction;
 
-	public SimpleAckingTaskManagerGateway() {
-		optSubmitConsumer = Optional.empty();
-		optCancelConsumer = Optional.empty();
-	}
+	@Nonnull
+	private volatile BiConsumer<InstanceID, Exception> disconnectFromJobManagerConsumer = (ignoredA, ignoredB) -> {};
```

Comment:
Where does this come from? Hmm it exists in master, so I suppose it's either a GH bug or something went wrong during a rebase.